src/main/cpp/auton/PrimitiveParser.cpp

```diff
@@ -71,6 +72,12 @@ PrimitiveParamsVector PrimitiveParser::ParseXML(string fulldirfile)
         {"NONE", ChassisOptionEnums::PathUpdateOption::NONE},
     };
 
+    map<string, DriveStopDelay::DelayOption> pathDelayOptionsMap{
+        {"START", DriveStopDelay::DelayOption::START},
+        {"PLACED_CORAL", DriveStopDelay::DelayOption::PLACED_CORAL},
```

Comment:
would have almost called this reef just to make it more location based like the other 2